test/data/err_155.py

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
These are good test cases, but they don't cover all the cases (ie, passing a string/byte literal, etc). To get to 100% test coverage, add the following tests/lines:

```python
from pathlib import Path

os.path.getsize("filename")
os.path.getsize(b"filename")
os.path.getsize(Path("filename"))

# this should not match
os.path.getsize(1)
```

The last line checks that Refurb *doesn't* emit an error for that line. Reason being is that a lot of the `os.path` functions take a file descriptor (as an integer), whereas the `Path` function cannot, and thus we have to ignore this edge case.

Basically add these test cases, rerun `make update-tests`, and ensure the lines look correct (and that the last line with `getsize(1)` isn't in there).